### DISCUSSION TITLE
A way to delete claims without standing on them

### DISCUSSION THREAD

**Discussion by Noxillio** [GriefPrevention/GriefPrevention#1845] [Ideas]
Make it so claims can be deleted remotely because players with 11 claims don’t want to have to fly around the map to get rid of them. A command, perhaps.

**Comment by xSepticSidx** [comment]
This would be good, the list of claims would then have to be numbered so the player can then choose the number of the claim to delete.

As someone that can often have claims in the high numbers to protect certain areas this would be useful.
As would then expanding it to enable commands to be run on remote claims such as:

First do /claimslist to identify the claim number by location
then i.e. use /accesstrust [player] [claimnumber]

Even better if claims can be named so that in the claimslist they have meaningful descriptions :)
